### PR TITLE
Enhance the README

### PR DESCRIPTION
**Pull request by satendrapalsingh**
> Rewrite the existing text to improve clarity
> Fix minor typos
> Add sections like  Installation, Usage, License, etc.

### CODE REVIEW

**Comment by sinkingsugar** [comment]
We need a full rewrite pivoting the future of this language as the foundation for building a game creation system. Not a grammar pass 😄

**Comment by satendrapalsingh** [comment]
> We need a full rewrite pivoting the future of this language as the foundation for building a game creation system. Not a grammar pass 😄

Yeah, I guessed as much😅. We will need to reassess the effort needed for this README issue #225.
@Kryptos-FR FYI.

**Comment by satendrapalsingh** [comment]
**Update**
- Branch rebased on top of devel to 
   - include language-renaming changes
   - retain new temporary logo
- Added some more points added to the **Features** section
- These changes and previous commits squashed into a single commit (with all commit messages retained) for easy review

File: README.md [hunk 1]
```diff
@@ -91,33 +108,28 @@ We introduce a low code way of producing high performance and multi-platform app
...
-## Motivation
-In all those years of software engineering I've been touching many fields including, audio, VR, physics, game engines in general but also machine learning, NNs etc... everything interesting and all but all the software I wrote didn't exactly connect to each other and connecting it would have been a big effort. I always wanted to mix things and recombine, experiment, after all I started as a musician using max/msp on my black powerbook pismo...
-
-Shards is my answer to this call, a tool that let's me write specific native code and connect it to any other code I wrote.
-
-Where **Wire** would be your procedure and **Shards** the building shards of your procedure.
-
-With a strong emphasis on automation and repeated tasks. Making each frame of execution of your script first class.
+```
+./cbl <filename.edn>
```
Comment (sinkingsugar) [inline README.md:128]: shards

Comment (satendrapalsingh) [reply README.md]: Fixed!

**Review by Kryptos-FR — APPROVED** [review]

**Comment by satendrapalsingh** [comment]
New logo updated

**Pull request merged**